### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by MSeechan**
Recoded with bootstrap and react routing. Separated the App.js to load multiple components with routing rather than loading 1 large file in sections.

### CODE REVIEW

**Comment by Ryan1Up** [comment]
Can you include the package.json file?

**Comment by Ryan1Up** [comment]
Who's Jason, and why did you add his files?!?!?

**Comment by Ryan1Up** [comment]
But does it work?

File: notifications/front_end/src/components/NavBarComp.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ const NavBarComp = ({authenticate}) => {
+  const navigate = useNavigate();
+  const signOut = () => {
+    authenticate();
+    navigate("/");
+    
+    
+};
+   
+  return (
```
Comment (Ryan1Up) [inline notifications/front_end/src/components/NavBarComp.js:15]: Is this supposed to be in NavBarComp? Or does the export statement execute this?

File: notifications/front_end/src/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      {/* Register Header */}
+      <div className="">
+        <div>
+          <h4>Sign In</h4>
+          <hr className="green"></hr>
+          <div className="var-box">Email</div> 
+          <div className="var-box">Password</div>
+          <div className="var-box"> Business Number</div>
+        </div>    
+
```
Comment (Ryan1Up) [inline notifications/front_end/src/SignIn.js:43]: Why no make it form?

```<form onSubmit={this.handleSubmit}}>
     <label>
          Email:
          <input type="text" value={this.state.value} onChange={this.handleChange} />
        </label>
        <label>
          Password:
          <input type="text" value={this.state.value} onChange={this.handleChange} />
        </label>
       <label>
          Business ID:
          <input type="text" value={this.state.value} onChange={this.handleChange} />
        </label>
        <input type="submit" value="Submit" />
      </form>
```

File: notifications/front_end/src/SignIn.js [hunk 3]
```diff
@@ -1,46 +1,55 @@
...
+          <h2>FR Most Wanted</h2>  
+        </div>    
+        
+            <h4>Sign In</h4>
+            <hr className="green"></hr>
+        
+            <Form.Group >     
+              <Form.Control className ="mb-3" type="email" placeholder="Enter email" />
+              <Form.Control className ="mb-3" type="email" placeholder="Password" />
+              <Form.Control className ="mb-3" type="email" placeholder="Business Number" />
```
[the remark this answers is not in the document]
Comment (MSeechan) [reply notifications/front_end/src/SignIn.js:37]: Noted: will update in the next commit

File: notifications/front_end/src/SignIn.js (hunk 2 above)
Comment (Ryan1Up) [reply notifications/front_end/src/SignIn.js:43]: disregard, i see the form

File: notifications/front_end/src/components/NavBarComp.js (hunk 1 above)
Comment (MSeechan) [reply notifications/front_end/src/components/NavBarComp.js:15]: The export grabs whatever is in the return and spits it out (I think that's what you're asking)

File: notifications/front_end/src/SignIn.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (MSeechan) [reply notifications/front_end/src/SignIn.js]: Updated

**Pull request merged**